root/programs/C#/Frameworks/Infrastructure/Business/Business/MyApsBaseLogic.cs

```diff
@@ -221,90 +221,97 @@ protected override void UOC_DoAction(BaseParameterValue parameterValue, ref Base
 
                 #region データ プロバイダ選択
 
-                if (parameterValue.ActionType.Split('%')[0] == "SQL")
-                {
-                    // SQL Server / SQL Client用のDamを生成
-                    damABT = new DamSqlSvr();
-                    damAMT = new DamSqlSvr();
+                // SQL Server / SQL Client用のDamを生成
+                damABT = new DamSqlSvr();
+                damAMT = new DamSqlSvr();
 
-                    // 接続文字列をロード
-                    connstring = GetConfigParameter.GetConnectionString("ConnectionString_SQL");
-                }
-                else if (parameterValue.ActionType.Split('%')[0] == "OLE")
-                {
-                    // OLEDB.NET用のDamを生成
-                    damABT = new DamOLEDB();
-                    damAMT = new DamOLEDB();
+                // 接続文字列をロード
+                connstring = GetConfigParameter.GetConnectionString("ConnectionString_SQL");
 
-                    // 接続文字列をロード
-                    connstring = GetConfigParameter.GetConnectionString("ConnectionString_OLE");
-                }
-                else if (parameterValue.ActionType.Split('%')[0] == "ODB")
-                {
-                    // ODBC.NET用のDamを生成
-                    damABT = new DamODBC();
-                    damAMT = new DamODBC();
+                //if (parameterValue.ActionType.Split('%')[0] == "SQL")
+                //{
+                //    // SQL Server / SQL Client用のDamを生成
+                //    damABT = new DamSqlSvr();
+                //    damAMT = new DamSqlSvr();
 
-                    // 接続文字列をロード
-                    connstring = GetConfigParameter.GetConnectionString("ConnectionString_ODBC");
-                }
-                else if (parameterValue.ActionType.Split('%')[0] == "ORA")
-                {
-                    // Oracle / Oracle Client用のDamを生成
-                    damABT = new DamOraClient();
-                    damAMT = new DamOraClient();
+                //    // 接続文字列をロード
+                //    connstring = GetConfigParameter.GetConnectionString("ConnectionString_SQL");
+                //}
+                //else if (parameterValue.ActionType.Split('%')[0] == "OLE")
+                //{
+                //    // OLEDB.NET用のDamを生成
+                //    damABT = new DamOLEDB();
+                //    damAMT = new DamOLEDB();
 
-                    // 接続文字列をロード
-                    connstring = GetConfigParameter.GetConnectionString("ConnectionString_ORA");
-                }
-                else if (parameterValue.ActionType.Split('%')[0] == "ODP")
-                {
-                    // Oracle / ODP.NET用のDamを生成
-                    damABT = new DamOraOdp();
-                    damAMT = new DamOraOdp();
+                //    // 接続文字列をロード
+                //    connstring = GetConfigParameter.GetConnectionString("ConnectionString_OLE");
+                //}
+                //else if (parameterValue.ActionType.Split('%')[0] == "ODB")
```

Comment:
@daisukenishino 

This program is only enabled for SQL Server.
To enable this program for other databases, uncomment these lines.